packages/by-chart/src/javascript/pi_chart.js

```
@@ -0,0 +1,35 @@
+      return d.value;
+    })(dataset);
+
+    vis
+      .selectAll("path")
+      .data(pie)
+      .enter()
+      .append("path")
+      .attr("d", arc)
+      .attr("fill", (d, i) => color(i));
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)